bootstrap/16-files.sh

```diff
@@ -0,0 +1,29 @@
+function install_recur() {
+    for i in "$1"/*
+    do
+        full_path=$i
+        relative_path=${i#$CUSTOM_FILES}
+        mode=`stat -c "%a" $full_path`
+
+        if [ -d "$full_path" ]; then
+            chroot_exec mkdir -p $relative_path
+            chroot_exec chmod $mode $relative_path
+
+            install_recur $full_path
+        elif [ -e "$full_path" ]; then
+            install -o root -g root -m $mode $full_path $R$relative_path
+        else
+            echo "${full_path} is empty"
```

Comment:
Use `info` instead of `echo`. `info`, `fatal` and `success` are wrappers around `echo` which output the specified text to stderr.